students/piotrowski_stanislaw/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,24 @@
+# piotrsta
+# Age calculator
+# Write a program that for given list of people's ages
+# calculate the average age of adults (age >= 18)
+# and count the children (age < 18). Use list comprehensions.
+
+
+def age_calculator(input_list):
+    adult_ages_list = [int(x) for x in input_list if 200 > int(x) >= 18]
+    child_ages_list = [int(x) for x in input_list if 0 < int(x) < 18]
+    if len(adult_ages_list) != 0:
```

Comment:
w takiej sytuacji lepiej uzywać zapisu:
`if adult_ages_list:` co działa jak `if len(adult_ages_list) > 0 and adult-ages_list is not None:'